03/app.js

```diff
@@ -0,0 +1,9 @@
+const btnsArr = document.querySelectorAll('button');
+const changeText = function (e) {
+	this.innerText = 'clicked';
+	console.log('clicked');
+};
+
+btnsArr.forEach((el) => {
+	el.addEventListener('click', changeText, { once: true });
```

Comment:
Trzeba uważać na IE: https://caniuse.com/once-event-listener
Spróbuj rozwiązać to zadanie tak, aby też działało na IE :)